tests/manage/pv_services/space_reclaim/test_auto_reclaim_space_cronjob.py

```diff
@@ -0,0 +1,223 @@
+    skipif_ocs_version,
+    ManageTest,
+    tier2,
+)
+from ocs_ci.utility.utils import run_cmd, TimeoutSampler
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers import helpers, performance_lib
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import PVCNotCreated, TimeoutExpiredError
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@tier2
+@skipif_ocs_version("<4.14")
+class TestReclaimSpaceCronJob(ManageTest):
+    """
+    Test that verifies automatic creation of Reclaim Space Cron Jobs for RBD PVCs in openshift-* namespaces
+    The test also verifies that no Reclaim Space Cron Job is created automatically for CephFS PVC in this namespace
+    """
+
+    def setup(self):
+        self.pvc_objs_created = []
+        self.temp_files_list = []
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Cleanup the test environment
+
+        """
+
+        def finalizer():
+            # pvc objects usually are deleted in the test (and the second delete has no effect),
+            # if for some reason the pvcs were not deleted in the test - they will be deleted here
+            for pvc_obj in self.pvc_objs_created:
+                pvc_obj.delete()
+
+            for temp_file in self.temp_files_list:
+                if os.path.exists(temp_file):
+                    run_cmd(f"rm {temp_file}")
+
+            run_oc_command(cmd=f"delete namespace {self.namespace}")
+
+        request.addfinalizer(finalizer)
+
+    def test_reclaim_space_cronjob(self):
+        """
+        Test case to check reclaim space cronjobs are created correctly for rbd pvcs in openshift-* namespace
+        """
+        num_of_pvcs = 10
+        namespace = f"openshift-{uuid4().hex}"
+        self.namespace = namespace
+        result = run_oc_command(cmd=f"create namespace {self.namespace}")
+        assert ERRMSG not in result[0], (
+            f"Failed to create namespace with name {namespace}" f"got result: {result}"
+        )
+        logger.info(f"Namespace {namespace} created")
+
+        result = run_oc_command(cmd=f"get namespace {namespace} -o yaml")
+        namespace_dict = yaml.safe_load("\n".join(result))
+
+        schedule = namespace_dict["metadata"]["annotations"][
+            "reclaimspace.csiaddons.openshift.io/schedule"
+        ]
+        assert (
+            schedule == "@weekly"
+        ), f"Namespace {namespace} created with schedule {schedule}, expected @weekly"
+        logger.info(f"Existence of schedule {schedule} validated.")
+
+        self.pvc_objs_created, _ = helpers.create_multiple_pvcs(
+            sc_name=constants.DEFAULT_STORAGECLASS_RBD,
+            namespace=self.namespace,
+            number_of_pvc=num_of_pvcs,
+            size="1Gi",
+        )
+
+        logger.info("Wait for all of the PVCs to be in Bound state")
+        performance_lib.wait_for_resource_bulk_status(
+            "pvc",
```

Comment:
this can be replaced with constant PVC

Reply:
Will be handled in my next PR

Reply:
Please explain what do you mean by "constant PVC" -- to print in the logger message the number of PVCs instead of 'all'?